### ISSUE TITLE
Permission denied on first run

### ISSUE THREAD

**Issue by Kixunil** [rust-embedded/rust-sysfs-gpio#24]
Hello, I've a problem in [Heimdall](https://github.com/Progressbar/heimdall). When I first run it, I get permission denied, but second time it works. I guess this is because export doesn't show up in /sys/ immediately. (I tried to unexport instead of rebooting and it panics too.)

Did you encounter this problem too? Is there any plan to fix it?

**Comment by posborne** [comment]
There's some discussion here: https://github.com/rust-embedded/rust-sysfs-gpio/issues/5.  I don't think there are any plans to handle this directly in this library.

**Comment by Kixunil** [comment]
OK, thank you. It might be useful to mention it in the API documentation though since I expect many people use the library with Raspberry Pi. (I didn't look at blinky example, just the API.) Also, it seems to be missing in README (which might be starting point for some people).

What do you think?

**Comment by posborne** [comment]
Yeah, would you be willing to open a PR with notes on possibly having to work around udev?
